ConsoleApp/Shop/Cotalog/Group.cs

```diff
@@ -0,0 +1,8 @@
+namespace ConsoleApp.Shop;
+using System.Collections.Generic;
+
+class Group
+{
+    private List<Group> _children { get; set; }
```

Comment:
code convention violated here
```suggestion
    private List<Group> Children { get; set; }
```